sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -416,9 +419,10 @@ def __str__(self):
     @classmethod
     def from_string(
         cls,
-        permission,  # type: str
+        permission,
         **kwargs
     ):
+        # Type: (str, **Dict[str]) -> AccountSasPermissions
```

Comment:
I think we don't actually need to type hint kwargs like this.
When I try to do so - VSCode gives me a squiggly yellow line.